### DISCUSSION TITLE
Unable to import paddle core

### DISCUSSION THREAD

**Discussion by clonesangram87** [PaddlePaddle/PaddleOCR#9989] [Q&A]
While running below code I am getting an error shown below. Please advise how to fix this? I am running it in Google Colab.

from paddleocr import PaddleOCR,draw_ocr
# Paddleocr supports Chinese, English, French, German, Korean and Japanese.
# You can set the parameter `lang` as `ch`, `en`, `fr`, `german`, `korean`, `japan`
# to switch the language model in order.
ocr = PaddleOCR(use_angle_cls=True, lang='en') # need to run only once to download and load model into memory

Error: Can not import paddle core while this file exists: /usr/local/lib/python3.10/dist-packages/paddle/fluid/libpaddle.so
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
[<ipython-input-8-959cb159bc2f>](https://localhost:8080/#) in <cell line: 1>()
----> 1 from paddleocr import PaddleOCR,draw_ocr
      2 # Paddleocr supports Chinese, English, French, German, Korean and Japanese.
      3 # You can set the parameter `lang` as `ch`, `en`, `fr`, `german`, `korean`, `japan`
      4 # to switch the language model in order.
      5 ocr = PaddleOCR(use_angle_cls=True, lang='en') # need to run only once to download and load model into memory

**Comment by seyed0123** [comment]
I also encountered this problem and no matter how much I looked for a solution in the official docs, I did not get any results

**Reply by sezerakr** [reply]
run this two commands

!wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4_amd64.deb
!sudo dpkg -i libssl1.1_1.1.0g-2ubuntu4_amd64.deb

**Reply by Bobivankenobi** [reply]
thanks bro, you saved me. 🥇

**Comment by washcycle** [comment]
downgrade to 2.6.1.3 it doesn't have that issue
